playbooks/roles/ocp-cli-plugins/tasks/install-cli-plugin-required-packages.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+## OCP CLI-plug-in required package installation module
+
+- name: Install prerequisite packages
+  yum:
+    name: "@Development tools"
+    state: latest
+
+- name: Install golang with version>=1.18
+  include_role:
+    name: golang-installation
+  vars:
+    go_tarball: "{{ golang_tar_url | default('https://go.dev/dl/go1.18.4.linux-ppc64le.tar.gz', true) }}"
```

Comment:
nit: It is better to add defaults in role level`defaults/main.yaml` then at the task level. Also could mention the default URL in the role doc.

Reply:
Added the tarball variable in the role  `defaults/main.yaml` and description in the new variable section for default variables